README.md

```diff
@@ -28,3 +28,57 @@ The Presto Foundation maintains a [public calendar](https://calendar.google.com/
 The TSC voting members are initially the [Project’s Committers](https://github.com/prestodb/presto/wiki/committers), and the current TSC chair is Nezih Yigitbasi ([@nezihyigitbasi](https://github.com/nezihyigitbasi)). At the inception of the project, the Committers of the Project will be as set forth within the "CONTRIBUTING" file within the Project’s code repository
 
 https://github.com/prestodb/presto/blob/master/CONTRIBUTING.md
+
+## Policies and procedures
+
+The Presto TSC is governed by the [Technical Charter](CHARTER.md).  The Charter provides a foundational structure for the TSC on topics such as its scope, how to make decisions, and how to make changes to itself.  At the same time, it grants the TSC a high degree of freedom when determining how to implement the policies of the Presto Foundation.
+
+The following policies and procedures have been adopted by the TSC.
+
+### Making decisions
+
+Per the [charter](CHARTER.md), wherever possible the TSC will attempt to make decisions by consensus.  In circumstances where consensus is not possible or if a vote is explicitly required, a majority (or higher, if required by the governance) of TSC voting members must approve in order for the action to proceed.  Votes will be taken over email, and documented in the next meeting.
+
+### Merging PRs into the TSC repository
+
+Pull requests that do not change the charter or governance of the TSC can be merged into this repository provided the following conditions have been met:
+
+* There are no outstanding objections
+* There are two approvals by TSC members
+* The PR has been open for at least 72 hours
+
+Pull requests that change governance of the TSC (excluding the charter) must be open for at least 14 days, unless consensus is reached in a meeting with quorum of voting members.
```

Comment:
Why only meetings and why not mailing list?

Reply:
Good question, I'll explain the rationale and we can figure out if it makes sense for Presto.

The idea here is that most things which modify policies or governance should be open for long enough that people who miss a call have enough time to review them (72 hours or 14 days, respectively).

On the other hand, if a meeting is held and there are enough voting members to make a decision on the spot, it wouldn't make sense to hold the PR any longer, because the result wouldn't change.